mathics_scanner/prescanner.py

```diff
@@ -59,7 +59,12 @@ def scan(self):
                     self.stubs.append(self.code[self.start : self.pos])
                     self.newstub(self.pos + 2)
                 else:
-                    self.pos += 1
+                    # If c == "\\", we have to avoid consider it as an escape
+                    # character in a way that, for instance r"...\\[Integral]..."
+                    # do not be parsed as r"...\\u222b...".
+                    # Any another character is not to be processed, so
```

Comment:
Based on what you report about lengths, this looks exactly correct.  Thanks! 

Suggested rewording of comment:

```
# Two backslashes in succession indicates a single backslash character, rather than an escape sequence which also starts with a backslash. 
#  Advance the scanning cursor (self.pos) over both backslashes.